master/buildbot/steps/trigger.py

```diff
@@ -63,7 +65,17 @@ def __init__(self, schedulerNames=None, sourceStamp=None, sourceStamps=None,
             config.error(
                 "You can't specify both alwaysUseLatest and updateSourceStamp"
             )
+        if len(set(schedulerNames) & set(unimportantSchedulerNames)) != len(unimportantSchedulerNames):
```

Comment:
there's an easier way to check this:
```python
if not set(schedulerNames).issuperset(set(unimportantSchedulerNames)):
 ...
```

(I'm using `superset` here to make it easier to read the code)